.circleci/config.yml

```diff
@@ -677,12 +677,11 @@ workflows:
       # Only runs on vX.X.X tags if all tests are green
       - publish_npm_package:
           filters:
-            branches:
-              only:
-                - /.*-stable/
             tags:
               only: /v[0-9]+(\.[0-9]+)*(\-rc(\.[0-9]+)?)?/
           requires:
+            - analyze
```

Comment:
Circle CI is more stable these days, let's avoid pushing a release when tests are red.